packages/chaire-lib-common/src/tasks/osrm/downloadOsmNetworkData.ts

```diff
@@ -68,6 +69,7 @@ export class DownloadOsmNetworkData implements GenericTask {
         // Make sure the number of retries is between 0 and 6
         const retryAttempts = isNaN(retry) ? 0 : Math.min(Math.max(retry, 0), 6);
         const importDir = this._fileManager.directoryManager.projectDirectory + '/imports/';
+        fs.mkdirSync(importDir, { recursive: true });
```

Comment:
I think it's the directory manager's role to make sure the project directory is created. It does not make sense that every possible task that can be run before the server first starts needs to make sure the directory exists. Besides, this is in `common` and you should not import fs here.

For info, currently, all directory creations are done at server start in the serverApp.ts file

Reply:
good point!
So serverApp.ts is already creating that directory: `directoryManager.createDirectoryIfNotExists('imports')`
So this PR is not necessary, I just had to launch Transition once before running the task.

Should I update the README to mention that?

Reply:
currently the steps in README.md are in the following order:
1. Create a config file
2. [Download and prepare the road network](https://github.com/chairemobilite/transition#download-and-prepare-the-road-network)
3. [Prepare the environment file](https://github.com/chairemobilite/transition#prepare-the-environment-file)
4. ...
5. [Start the Node.js server](https://github.com/chairemobilite/transition#start-the-nodejs-server)

so it seems like step 5 should be done before step 2?

Reply:
I don't think we should have to run transition before the task (I run the task independently in some of our infra. 
I don't know if it should create the directly, but the task should at least fail cleanly, so it should at least check if the directory is present and print an error message saying that the directly is invalid and suggest appropriate steps.